pygments/styles/material.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+"""
+    pygments.styles.material
+    ~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Mimic the Material theme color scheme.
+
+    https://github.com/material-theme/vsc-material-theme
+
+    :copyright: Copyright 2006-2021 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+from pygments.style import Style
+from pygments.token import Keyword, Name, Comment, String, Escape, \
+    Error, Text, Number, Operator, Generic, Punctuation, Literal
+
+class MaterialStyle(Style):
+    """
+    This style mimics the Material Theme color scheme.
+    """
+    dark_teal = '#263238'
+    white= '#FFFFFF'
+    black= '#000000'
+    red= '#FF5370'
+    orange= '#F78C6C'
+    yellow= '#FFCB6B'
+    green= '#C3E88D'
+    cyan= '#89DDFF'
+    blue= '#82AAFF'
+    paleblue= '#B2CCD6'
+    purple= '#C792EA'
+    brown= '#C17E70'
+    pink= '#F07178'
+    violet= '#BB80B3'
+    foreground = '#EEFFFF'
+    faded = '#546E7A'
+
+    default_style = ""
+    background_color = dark_teal
+    highlight_color = '#507174'
+    line_number_color = '#37474F'
+    line_number_background_color = dark_teal
+    line_number_special_color = faded
```

Comment:
Can you please set `line_number_special_background_color` as well?

Reply:
Good catch! Done!